UnitsConverter.Fluent.Plugin/ConvertUtil.cs

```diff
@@ -27,6 +27,82 @@ internal class ConvertUtil
             QuantityType.Volume,
         };
 
+        public static List<IQuantity> ConvertWithoutQuantity(string text)
```

Comment:
I suggest making this `bool TryConvertWithQuantity(string text, out List<IQuantity> quantity)` and then inline it with the `if` in the search app, line 113.

Reply:
I see, you prefer returning the bool instead of using empty list to represent conversion failure?